src/Castle.Windsor.Extensions.DependencyInjection/Scope/ExtensionContainerScope.cs

```diff
@@ -20,36 +20,37 @@ namespace Castle.Windsor.Extensions.DependencyInjection.Scope
 	using Castle.Core;
 	using Castle.MicroKernel;
 	using Castle.MicroKernel.Lifestyle.Scoped;
-	
+
 	internal class ExtensionContainerScope : ILifetimeScope, IDisposable
 	{
-		public static ExtensionContainerScope Current => current.Value;
+		internal static ExtensionContainerScope Current
+		{
+			get => current.Value;
+			set => current.Value = value;
+		}
 		public static string TransientMarker = "Transient";
-		protected static readonly AsyncLocal<ExtensionContainerScope> current = new AsyncLocal<ExtensionContainerScope>();
+		protected static readonly AsyncLocal<ExtensionContainerScope> current  = new AsyncLocal<ExtensionContainerScope>();
```

Comment:
Double space

Reply:
> Sorry @generik0 for the back and forth on this pull request, I've got a few more review comments on the recent commits.

@jonorossi  I am happy for the reviews. I feel the change is getting better and better due to the good reviews. thankyou.

Reply:
fixed